client/reducers/login.js

```diff
@@ -7,10 +7,15 @@ const initialState = {
 export default (state = initialState, action) => {
     switch (action.type) {
         case CHECK_LOGGED_IN: {
+            const { username, name, email, id, roles } = action.payload;
             return {
                 ...state,
                 isLoggedIn: true,
-                ...action.payload
```

Comment:
Revert to `...action.payload` to remove the bug. Otherwise, cannot save the user's ATs